errata_tool/release.py

```diff
@@ -54,6 +54,20 @@ def refresh(self):
         # For displaying in scripts/logs:
         self.edit_url = self._url + '/release/edit/%d' % self.id
 
+    def get_product_versions(self):
+        url = self._url + '/api/v1/releases' % self.id
```

Comment:
As it turns out, when we make the main API request in the `refresh()` method above, the Errata Tool sends us back some JSON that also includes the product version information. 

We weren't saving it, because when I initially wrote this class, it wasn't available for PDC-based releases! But PDC is gone now :)

https://github.com/red-hat-storage/errata-tool/pull/126 populates the `.product_versions` attribute. Are you ok with that approach instead?

Reply:
Wau. I am ok, with that.